README.MD

```diff
@@ -1,211 +1,121 @@
-# gene-benchmark
+# Gene Benchmark
 
-A benchmark for gene embeddings from pretrained models.
+A benchmark for gene downstream tasks.
 
-The package includes three main sections:
+The repository is divided into the following sections:
 
-1. Descriptions - Retrieval of textual descriptions from pre existing files or database.
-2. Encode  the textual data
-3. Tasks - Evaluation of the efficacy of these encodings in downstream tasks.
+* [gene_benchmark](./gene_benchmark/): The package itself, containing the scripts for extracting textual descriptions from NCBI, encoding textual descriptions and evaluation of model performance.
 
-## Description
+* [notebooks](./notebooks/): Notebooks for creating the results figures and package usage examples.
 
-Retrieval of textual descriptions from pre existing files or NCBI. Currently, the package can extract description for a single entity type (for example, gene symbols ..) or for multiple entity types (for example, gene symbols and disease...)
+* [scripts](./scripts/): Scripts for description extraction, encoding, task creation and execution.
 
-### Single entity type descriptions
+* [tasks](./tasks/): The default directory that will be populated with all the tasks after running the task creation script.
 
-Currently, the descriptions can be retrieved from the NCBI database and from a pre existing CSV with the text descriptions.
+In depth explanation on each of the packages components can be found in the `gene_benchmark` directory.
 
```

Comment:
Even before the environment add a code snippet that assumes that the package is installed and set-up (explicitly say that) and runs all the binary tasks on a huggingface model / assuming that you already have an embedding file. This should 3-4 lines of code, showing how easy it is to use the package.